.github/workflows/github-status-test-lambda.yml

```diff
@@ -0,0 +1,42 @@
+name: github-status-test-lambda.yml
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/github-status-test-lambda.yml
+      - aws/lambda/github-status-test/**
+
+defaults:
+  run:
+    working-directory: aws/lambda/github-status-test/
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+    steps:
+      - name: Check out test infra
+        uses: actions/checkout@v2
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+
+      - uses: nick-fields/retry@71062288b76e2b6214ebde0e673ce0de1755740a
+        name: Setup dependencies
+        with:
+          shell: bash
+          timeout_minutes: 10
+          max_attempts: 3
+          retry_wait_seconds: 30
+          command: |
+            set -eux
+            python3 -m pip install awscli
+
+      - name: Deploy the lambda
```

Comment:
Might be worth linking to the readme here as well so folks realize the "deploy" step doesn't actually handle the end to end deployment

Reply:
Sounds good.  My overall thinking is that is sounds dangerous to deploy the lambda to prod right away after committing to main without any form of testing.  Previously, people can go to AWS lambda and update the code directly there, which is definitely not good neither.  So the middle option is to automate most of the steps for now and requires manual verification at the final step.  We can add more tests and guardrails later to make the deployment fully automated like `log_classifier` lambda for example, but it's kind of low priority because this lambda has rarely been updated.